
            <!DOCTYPE html>
            <html lang="en">
                <head>
                    <meta charset="UTF-8" />
                    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
                    <title>Schema Evaluator - Individual Analysis</title>
                </head>
                <body style="background-color: #f6f6f6">
                    <noscript> You need to enable JavaScript to run this app. </noscript>
                    <div id="root"></div>
                    <script> const _preComputedDataInd = {"data": {"allele_sizes": [237, 237, 240, 240, 243, 252], "alleles_median": 240, "alleles_mode": 237, "locus_ids": ["GCF-001275805-protein166_1", "GCF-001275805-protein166_S_GCF-000695215.1-ASM69521v1.fasta_17/04/2021T18:08:09_2", "GCF-001275805-protein166_S_GCF-003186125.1-ASM318612v1.fasta_17/04/2021T18:08:25_3", "GCF-001275805-protein166_S_GCF-016413705.1-ASM1641370v1.fasta_17/04/2021T18:08:50_4", "GCF-001275805-protein166_S_GCF-016894285.1-ASM1689428v1.fasta_17/04/2021T18:08:50_5", "GCF-001275805-protein166_S_GCF-900324205.1-1-1439.fasta_17/04/2021T18:08:52_6"], "nr_alleles": 6, "size_range": "237-252"}, "locus_name": "GCF-001275805-protein166"} </script>
                    <script> const _exceptions = [] </script>
                    <script> const _cdsDf = {"Alleles not multiple of 3": 0, "Alleles shorter than 201 nucleotides": 0, "Alleles w/ >1 stop codons": 0, "Alleles wo/ Start/Stop Codon": 0, "CDS": 6, "Gene": "GCF-001275805-protein166.fasta", "Missing Allele IDs": ["None"], "Number of alleles": 6, "Total Invalid Alleles": 0, "coding_strand": "antisense", "contig": "NZ_LITH01000009.1", "genome": "GCF_001275805", "name": "Helix-turn-helix transcriptional regulator", "proteome_bsr": "-", "proteome_gene_name": "-", "proteome_id": "-", "proteome_product": "-", "proteome_species": "-", "start": "3554", "stop": "3806", "url": "http://purl.uniprot.org/uniprot/A0A5C8X7E0"} </script>
                    <script> const _msaData = {"sequences": [{"name": "1", "sequence": "MNENVKLLLRAARINNGMSQKKAASLLGVHYQTLASWEKDSSDLSRSKMLLMEKIYSIPLEYIFLVMKTSLFVQK-EGLNEPIT"}, {"name": "2", "sequence": "MNENVKLLLRAARINNGMSQKKAASLLGVHYQTLASWEKDSSDLSRSKMLLMEKIYSIPLEYIFFGNENEFIRTKRRG------"}, {"name": "3", "sequence": "MNENVKLLLRAARINNGMSQKKAASLLGVHYQTLASWEKDSSDLSRSKMLLMEKIYSIPLEYIFFGNENEFIRTK--GGVK---"}, {"name": "4", "sequence": "MNENVKLLLRAARINNGMSQKKAASLLGVHYQTLASWEKDSSDLSRSKMLLMEKIYSIPLEYIFFGNENEFIRTKRRG------"}, {"name": "5", "sequence": "MNENIKLLLRAARINNGMSQKKAASLLGVHYQTLASWEKDSSDLSRSKMLLMEKIYGIPLEYIFFGNENEFIRIK-RGRIK---"}, {"name": "6", "sequence": "MNENVKLLLRAARINNGMSQKKAASLLGVHYQTLASWEKDSSDLPRSKMLLMEKIYSIPLEYIFFGNENEFIRTK--GGVK---"}]} </script>
                    <script> const _phyloData = {"phylo_data": "(\n1_1\n:0.17645,(((\n2_2\n:0.00000,\n4_4\n:0.00000):0.03733,(\n3_3\n:0.00000,\n6_6\n:0.00000):0.03733):0.02553,\n5_5\n:0.06286):0.11359);\n"} </script>
                    <script> const _minLen = 201 </script>
                    <script src="./main_ind.js"></script>
                </body>
            </html>
            